docs/USER_GUIDE.md

```diff
@@ -163,6 +167,12 @@ resources.
      this stack in CloudFormation once deployed.
    - **AdminEmail:** The email address you wish to setup as the initial user of
      this Amazon S3 Find and Forget deployment.
+   - **DeployWebUI:** (Default: true) Choose whether to deploy the Web UI as
+     part of the solution. If you do deploy the Web UI AuthMethod must use
+     Cognito. If you do not use the Web UI interaction with the solution is done
+     through the APIs only.
+   - **AuthMethod:** (Default: Cognito) Choose which method of authentication
```

Comment:
have you tested switching from cognito -> sigv4 and vice versa?

Reply:
I have added a Note on line 177 of this file now to address this.